app/models/arclight/document_downloads.rb

```diff
@@ -3,28 +3,35 @@
 module Arclight
   ##
   # Model the Download links that can be configured (via YAML) for a collection
-  class CollectionDownloads
+  # or container
+  class DocumentDownloads
```

Comment:
@camillevilla Just as a heads up this has implications on #750

Reply:
@jkeck Do you have any thoughts on the name of this?

Reply:
I'm 👍 to `Document`.